core/src/test/java/hudson/PluginManagerTest.java

```diff
@@ -85,7 +85,7 @@ public void parseInvalidRequestedPlugins() throws Exception {
                 () -> pluginManager.parseRequestedPlugins(new ByteArrayInputStream(evilXML.getBytes(StandardCharsets.UTF_8))),
                 "XML contains an external entity, but no exception was thrown.");
         assertThat(ex.getCause(), instanceOf(SAXException.class));
-        assertThat(ex.getCause().getMessage(), containsString("DOCTYPE is disallowed"));
+        assertThat(ex.getCause().getMessage(), containsString("DOCTYPE"));
```

Comment:
Why not also check that the message contains the URL, as that is the same in all translations?

```suggestion
        assertThat(ex.getCause().getMessage(), containsString("DOCTYPE"));
        assertThat(ex.getCause().getMessage(), containsString("http://apache.org/xml/features/disallow-doctype-decl"));
```